### DISCUSSION TITLE
what happened?

### DISCUSSION THREAD

**Discussion by canadamadman** [BitMaker-hub/NerdMiner_v2#438] [General]
where did all the bin files go for all the different boards that were working?

**Comment by jsy-rg1570** [comment]
Dear, check releases and https://flasher.bitronics.store/ please 😉

**Reply by canadamadman** [reply]
so how do i download the files?

**Reply by jsy-rg1570** [reply]
You can download them on GitHub releases or pre-releases or flash your board with the webui with GitHub files.

**Reply by jsy-rg1570** [reply]
For example : https://github.com/BitMaker-hub/NerdMiner_v2/releases/tag/nerdminer-prerelease-V1.7.0

**Reply by canadamadman** [reply]
the link on the nerdminer web page need to be changes as it links to a page with no bin files

**Reply by jsy-rg1570** [reply]
@canadamadman can you show @BitMaker-hub what link should be changed please ?

**Reply by canadamadman** [reply]
![Screenshot 2024-06-27 170323](https://github.com/BitMaker-hub/NerdMiner_v2/assets/113493462/ecd9082e-1df4-4acb-bd19-a796e33066e2)

this link goes to a git hub with an empty bin folder

**Reply by BitMaker-hub** [reply]
Hi, sorry, I don't have powers of that website. Even it seems strange, I don't own NerdMiner.com so please tell them to change it.
